Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM gradle:8.10.2-jdk17 as build
+
+WORKDIR /app
+
+COPY . .
```

Comment:
To avoid copying everything from the project, what about adding a `.dockerignore` file to our project?

Reply:
done